qa/L1_superres_pytorch/test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
+pip_packages="numpy torch torchvision scikit-image tensorboardX"
+
+apt-get update
+apt-get install -y wget ffmpeg git
+
+pushd ../..
+
+source qa/setup_dali_extra.sh
+
+cd docs/examples/video
+
+mkdir -p video_files
+
+container_path=${DALI_EXTRA_PATH}/db/optical_flow/sintel_trailer/sintel_trailer.mp4
+
+IFS='/' read -a container_name <<< "$container_path"
+IFS='.' read -a splitted <<< "${container_name[-1]}"
+
+for i in {0..4};
+do
+    ffmpeg -ss 00:00:${i}0 -t 00:00:10 -i $container_path -vcodec copy -acodec copy -y video_files/${splitted[0]}_$i.${splitted[1]}
+done
+
+cd superres_pytorch
+
+DATA_DIR=data_dir/720p/scenes
+# Creating simple working env for PyTorch SuperRes example
+mkdir -p $DATA_DIR/train/
+mkdir -p $DATA_DIR/val/
+
+
+cp ../video_files/* $DATA_DIR/train/
+cp ../video_files/* $DATA_DIR/val/
+
+# Pre-trained FlowNet2.0 weights
```

Comment:
Could you add the reference how to get this file manually?

Reply:
The pre-trained model on the CI is actually taken from the publicly available on Google Drive https://drive.google.com/file/d/1QW03eyYG_vD-dT-Mx4wopYvtPu_msTKn/view and the link is in the [README here](https://github.com/NVIDIA/DALI/blob/0de4db3a8ad72ea6fbf6559f2c6a3bd97b789307/docs/examples/video/superres_pytorch/README.rst#flownet2-sd-implementation-and-pre-trained-model)

Reply:
I know, but if someone starts redoing tests may be not aware of this.

Reply:
Sure, just added the comment